cnxepub/tests/data/desserts-includes-py2.xhtml

```diff
@@ -1,6 +1,18 @@
 <html
   lang='en'
   xmlns='http://www.w3.org/1999/xhtml'
+  xmlns:bib='http://bibtexml.sf.net/'
+  xmlns:c='http://cnx.rice.edu/cnxml'
+  xmlns:data='http://www.w3.org/TR/html5/dom.html#custom-data-attribute'
+  xmlns:datadev='http://dev.w3.org/html5/spec/#custom'
+  xmlns:dc='http://purl.org/dc/elements/1.1/'
+  xmlns:epub='http://www.idpf.org/2007/ops'
+  xmlns:lrmi='http://lrmi.net/the-specification'
+  xmlns:m='http://www.w3.org/1998/Math/MathML'
+  xmlns:md='http://cnx.rice.edu/mdml'
+  xmlns:mod='http://cnx.rice.edu/#moduleIds'
+  xmlns:qml='http://cnx.rice.edu/qml/1.0'
+  xmlns:rdf='http://www.w3.org/1999/02/22-rdf-syntax-ns#'
```

Comment:
This is a regression, no?

Reply:
I don't think that was exactly what we were doing with `_fix_namespace`.  We were trying to put all the namespace in the root tag.  I think I can use `register_namespace` and this together ... :thinking:

Reply:
Ok, I think I've fixed it now!